cosmo_tester/framework/test_hosts.py

```diff
@@ -698,7 +701,9 @@ def __init__(self, *args, **kwargs):
         super(BootstrapBasedCloudifyManagers, self).__init__(*args, **kwargs)
         for manager in self.instances:
             manager.image_name = self._attributes.centos_7_image_name
-        self._manager_install_rpm = util.get_manager_install_rpm_url()
+
+        self._manager_install_rpm = ATTRIBUTES.cloudify_manager_install_rpm_url.strip() \
```

Comment:
I'm guessing this is what you were asking me about. As far as I can see, it's not used anywhere, so I suggest removing it.